catalog/resource_sql_table.go

```diff
@@ -171,7 +205,22 @@ func (ti *SqlTableInfo) getOrCreateCluster(clusterName string, clustersAPI clust
 	return aclCluster.ClusterID, nil
 }
 
+func (ci *SqlColumnInfo) getColumnType() string {
+	var colType string
+	switch ci.Identity {
+	case IdentityColumnAlways:
+		colType = "BIGINT GENERATED ALWAYS AS IDENTITY"
```

Comment:
Is the colType always BIGINT when using an identity column? What about if a user specified a different type?